packages/utils/src/loaders.ts

```diff
@@ -98,6 +104,38 @@ export async function getTranslations<L extends string = Languages>(
   return translationsArrayToObject(translationsArray);
 }
 
+async function populatePartialsObject(
+  partialsMap: Record<string, string>,
+): Promise<Record<string, TemplateDelegate<any>>> {
+  const partials = {};
+
+  for (const partial in partialsMap) {
+    const loadedTemplate = await loadExternalFile(partialsMap[partial], 'text');
+    partials[partial] = (ctx, opts) => compile(loadedTemplate)(ctx, opts);
+  }
+
+  return partials;
+}
+
+export async function getPartials(
+  partialsMap: Record<string, string>,
```

Comment:
base on your code logic, partialsMap should not be required in the params.
```ts
partialsMap?:  Record<string, string>,
```
would suit better.
Also in that case it should be the second argument, as non required arguments are positioned at the right most position. In fact we might even have an ESLint rule that should warn you about that.